Docs/Sample.plist

```diff
@@ -1156,32 +1158,8 @@
 		</dict>
 		<key>Serial</key>
```

Comment:
I removed the `Override` option here too. However, OcSerializeLib will complain now. Is it possible to make BOOLEAN optional as well?

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/8c965b6bd6813e3531f3d422edd16fdf97fca792